kevm-pyk/src/kevm_pyk/kevm.py

```diff
@@ -369,3 +371,58 @@ def accounts(accts: list[KInner]) -> KInner:
             else:
                 wrapped_accounts.append(acct)
         return build_assoc(KApply('.AccountCellMap'), KLabel('_AccountCellMap_'), wrapped_accounts)
+
+    def pretty_print(self, kast: KAst, *, unalias: bool = True, sort_collections: bool = False) -> str:
```

Comment:
What's still missing is testing that a `KOuter` is printed as expected.

This function returns a `str`, so a better way to organize it would be:

1. Transform `KAst -> KAst`. You can write robust tests for this.
2. Call `super().pretty_print` on the transformed `KAst`. This is tested in `pyk`, so if the `str` output changes (e.g. we do not print as many whitespace characters), tests are updated there.

Right now, these two steps are intertwined in a non-trivial recursion scheme.

If you do not address this in this PR (I do not expect you to), please open an issue to mark this request.